data/json/npcs/TALK_FACTION_CAMP.json

```diff
@@ -16,7 +16,7 @@
         "text": "I want you to build a camp here.",
         "topic": "TALK_DONE",
         "effect": "start_camp",
-        "condition": { "and": [ { "u_at_om_location": "field" }, { "npc_allies": 2 } ] },
+        "condition": { "and": [ { "u_at_om_location": "field" }, { "npc_allies": 1 } ] },
```

Comment:
I think you need to remove the switch and default lines as well and talking to an npc already requires 1 ally, so it should be like this:
```diff
-       "condition": { "and": [ { "u_at_om_location": "field" }, { "npc_allies": 2 } ] },
-       "switch": true,
-       "default": true
+       "condition": { "u_at_om_location": "field" }
```